tools/server/webui/src/lib/services/chat.ts

```diff
@@ -256,10 +256,8 @@ export class ChatService {
 		}
 
 		const decoder = new TextDecoder();
-		let fullResponse = '';
+		let aggregatedContent = '';
```

Comment:
I think we could have a more consistent naming for these variables... @ServeurpersoCom I'm open to your suggestions :)